script.js

```diff
@@ -0,0 +1,6 @@
+const mobileMenu = document.querySelector('.icon');
+const menuHeader = document.querySelector('.header');
+const toggleMenu = () => {
+  menuHeader.classList.toggle('active');
+};
+mobileMenu.addEventListener('click', () => toggleMenu());
```

Comment:
- [ ] Your mobile menu is working almost perfectly. But when the user clicks the menu option, he/she is correctly redirected to the part of the page but the mobile menu does not disappear. Kindly make it disappear when the user clicks on any link.
 - [ ] Your mobile menu close button does not appear when the menu is opened. Please make it appear so the user can see a way to close the menu.
 - Mobile menu:
 - 
<img width="446" alt="image" src="https://user-images.githubusercontent.com/61919617/196981579-02235d2b-7637-425c-bbb0-5357f845e632.png">